src/utils/socket.ts

```diff
@@ -52,25 +51,24 @@ export function addListeners(
       // TODO decide what to do depending on error. For now we just throw them
       // to create
       // TODO snackbar
-      console.log(message + '... creating new room');
+      console.log(`received ${MALFORMED_MESSAGE_RESPONSE}`, message);
       history.push('/duet');
     }
   );
   socket.on(UNKNOWN_MESSAGE_RESPONSE, ({ error }: UnknownErrorResponse) => {
     // TODO decide what to do depending on error. For now we just throw them
     // to create
     // TODO snackbar
-    console.log(error + '... creating new room');
+    console.log(`received ${UNKNOWN_MESSAGE_RESPONSE}`, error);
     history.push('/duet');
   });
 
   /********************* Join room ****************/
   socket.on(JOIN_ROOM_RESPONSE, (res: JoinRoomResponse) => {
     if (!res.success) {
-      // failed to join room, so create a room instead
+      // failed to join room, redirect back to duet page
       // TODO snackbar
-      const { code, message } = res as JoinRoomFailureResponse;
-      console.log(code + ': ' + message + '... creating new room');
+      console.log('failed to join room', res);
```

Comment:
can we display a notif here as well? Feels very abrupt right now? Or alternatively only transition to the lobby view if the join is successful

Reply:
hmm there's a TODO above